index.bs

```diff
@@ -2644,6 +2664,9 @@ This [=registration extension=] and [=authentication extension=] enables use of
 : Client processing
 :: None, except default forwarding of client argument to authenticator argument.
 
+: Client data
+:: Returns a JSON array of 3-element arrays of numbers that encodes the factors in the authenticator data
+
```

Comment:
ClientData (which must be hashed and fed into the authenticator for signature) cannot depend on AuthenticatorData (which comes out of the authenticator after the signature has been computed). Should be "none".